linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml

```diff
@@ -17,9 +17,13 @@ rule:
       - basic block:
         - and:
           - mnemonic: add
-          - instruction:
-            - mnemonic: imul
-            - operand[2].number: 0x801
+          - or:
+            - instruction:
+              - mnemonic: imul
```

Comment:
```suggestion
            - instruction:
              - or:
                - mnemonic: mul
                - mnemonic: imul
```

Reply:
this should work and you could remove the other addition

Reply:
But then I'd need to remove following `operand[2].number: 0x801` and replace it with just `number: 0x801` as `mul` does not take immediate values as operands?

Reply:
ah, my bad, your suggestion as is, is great!